sdk/src/main/java/com/silanis/esl/sdk/Signer.java

```diff
@@ -343,4 +344,12 @@ public KnowledgeBasedAuthentication getKnowledgeBasedAuthentication() {
     public void setKnowledgeBasedAuthentication(KnowledgeBasedAuthentication knowledgeBasedAuthentication) {
         this.knowledgeBasedAuthentication = knowledgeBasedAuthentication;
     }
+
+    public String getLocalLanguage() {
+        return localLanguage;
+    }
+
+    public void setLocalLanguage(String localLanguage) {
+        this.localLanguage = localLanguage;
+    }
```

Comment:
Do we need this local language field to use the local language? We set the local language as data in Role.
Why do we need duplicate settings?